apps/src/bin/pravega_event_dumper.rs

```diff
@@ -0,0 +1,117 @@
+//
+// Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
```

Comment:
License missing copyright year. Same for `pravega_event_dumper.sh`

Reply:
We may add a  jira story to fix this for all the files in this repo. For now, just to align with the other files.